packages/math/src/ObservablePoint.ts

```diff
@@ -117,10 +124,8 @@ export class ObservablePoint<T = any> implements IPoint
     }
     // #endif
 
-    /**
-     * The position of the displayObject on the x axis relative to the local coordinates of the parent.
-     *
-     * @member {number}
+    /** Position of the observable point on the x axis
+     * @type {number}
```

Comment:
Again, needed to explicitly set the type of this member. It wasn't inferring on either getter or setter.